src/actions.js

```diff
@@ -301,6 +312,23 @@ export const startWallet = (words, pin) => async (dispatch) => {
   // Set useWalletService on the redux store
   dispatch(setUseWalletService(useWalletService));
 
+  const showPinScreenForResult = async () => new Promise((resolve) => {
+    const params = {
+      cb: (_pin) => {
+        dispatch(setIsShowingPinScreen(false));
+        resolve(_pin);
+      },
+      canCancel: true,
```

Comment:
What does this mean? Should we `setIsShowingPinScreen(false)` if this is cancelled as well?

Reply:
We could pass a callback to `showingPinScreen` to false, but I think that we should force the user to input his PIN so his token is refreshed

Done in 26155acad255c2a0b45acfc291c9531212b72f3f

What do you think?